options.c

```diff
@@ -2524,6 +2524,16 @@ struct fio_option fio_options[FIO_MAX_OPTS] = {
 #endif
 		},
 	},
+	{
+		.name	= "prepopulate",
+		.lname	= "Prepopulate",
+		.type	= FIO_OPT_BOOL,
+		.off1	= offsetof(struct thread_options, prepopulate),
+		.help	= "Prepopulate files with some data",
+		.def	= "0",
+		.category = FIO_OPT_C_FILE,
+		.group	= FIO_OPT_G_INVALID,
```

Comment:
I'm really not sure if the group was picked properly, pls verify

Reply:
I'd agree with this because things like `fallocate()` are similar.